auth-web/package.json

```diff
@@ -20,7 +20,7 @@
     "@bcrs-shared-components/corp-type-module": "1.0.16",
     "@bcrs-shared-components/enums": "1.1.10",
     "@bcrs-shared-components/interfaces": "1.1.13",
-    "@mdi/font": "^4.5.95",
+    "@mdi/font": "^5.9.55",
```

Comment:
@ochiu Ody, I noticed this change was reverted at one point: https://github.com/bcgov/sbc-auth/pull/2716

What kind of issues were you running into exactly? I'm thinking I need to change this because I need the `mdi-delete-clock` icon in the CardHeader (line 45 in the `InvoluntaryDissolution.vue`). Please let me know!

Reply:
@JazzarKarim I believe there were icons we are currently using that have been removed in newer versions, so I had to revert the changes as it affected other screens.

Reply:
Example:

Your PR:
![Screenshot 2024-05-07 at 11 23 50 AM](https://github.com/bcgov/sbc-auth/assets/5702212/c62825b2-3c04-4c0e-987c-8af9fed1314b)

Dev:
![Screenshot 2024-05-07 at 11 24 20 AM](https://github.com/bcgov/sbc-auth/assets/5702212/45ff287d-6a85-4a7e-8244-f36f984fd0ad)

Reply:
I see, OK. Got it. Thank you so much Ody. 

I will talk to our UX designer and will see if we can use something else in this case. Appreciate it!

Reply:
You can also pull in icons separately if the font doesn't have them.

Reply:
Reverted this change to not break other stuff as Ody mentioned.